vite.config.ts

```diff
@@ -0,0 +1,72 @@
+import { defineConfig, PluginOption } from 'vite'
+import react from '@vitejs/plugin-react-swc'
+import viteTsconfigPaths from 'vite-tsconfig-paths'
+import svgrPlugin from 'vite-plugin-svgr'
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
+import { lingui } from '@lingui/vite-plugin'
+import checker from 'vite-plugin-checker'
+import dynamicImports from 'vite-plugin-dynamic-import'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+import fs from 'fs'
+// import { visualizer } from 'rollup-plugin-visualizer'
+
+// https://vitejs.dev/config/
+
+let https: any
+if (process.env.HTTPS === 'true') {
+    https = {
+        key: fs.readFileSync(process.env.SSL_KEY_FILE as any),
+        cert: fs.readFileSync(process.env.SSL_CRT_FILE as any)
+    }
+} else {
+    https = false
+}
+export default defineConfig({
+    envPrefix: 'REACT_APP_',
+    server: {
+        https
+    },
+    plugins: [
+        // visualizer({
+        //     template: 'treemap', // or sunburst
+        //     open: true,
+        //     gzipSize: true,
+        //     brotliSize: true,
+        //     filename: 'analice.html',
+        // }) as PluginOption,
+        dynamicImports(), //for lingui dynamic import lang files
+        // checker({
+        //     // e.g. use TypeScript check
+        //     typescript: true,
```

Comment:
why ts check is disabled ?

Reply:
it takes long time. i prefer typechecking is part of commit lint-stage and in code editor